file/mix.exs

```diff
@@ -0,0 +1,102 @@
+defmodule Membrane.RTC.Engine.Endpoint.File.MixProject do
+  use Mix.Project
+
+  @version "0.3.0-dev"
+  @engine_github_url "https://github.com/jellyfish-dev/membrane_rtc_engine"
+  @github_url "#{@engine_github_url}/tree/master/file"
+  @source_ref "file-v#{@version}"
+
+  def project do
+    [
+      app: :membrane_rtc_engine_file,
+      version: @version,
+      elixir: "~> 1.14",
+      elixirc_paths: elixirc_paths(Mix.env()),
+      start_permanent: Mix.env() == :prod,
+      deps: deps(),
+
+      # hex
+      description: "File Endpoint for Membrane RTC Engine",
+      package: package(),
+
+      # docs
+      name: "Membrane RTC Engine File Endpoint",
+      source_url: @github_url,
+      homepage_url: "https://membrane.stream",
+      docs: docs(),
+
+      # test coverage
+      test_coverage: [tool: ExCoveralls],
+      preferred_cli_env: [
+        coveralls: :test,
+        "coveralls.detail": :test,
+        "coveralls.post": :test,
+        "coveralls.html": :test,
+        "coveralls.json": :test
+      ],
+
+      # dialyzer
+      # this is because of optional dependencies
+      # they are not included in PLT
+      dialyzer: [
+        plt_add_apps: [
+          :membrane_video_compositor_plugin
+        ]
+      ]
+    ]
+  end
+
+  def application do
+    [
+      extra_applications: []
+    ]
+  end
+
+  defp elixirc_paths(:test), do: ["lib", "test/support"]
+  defp elixirc_paths(_env), do: ["lib"]
+
+  defp deps do
+    [
+      # Engine deps
+      {:membrane_rtc_engine, path: "../engine"},
+      {:membrane_rtc_engine_webrtc, path: "../webrtc"},
+
+      # Regular deps
+      {:membrane_core, "~> 0.12.3"},
+      {:membrane_aac_plugin, "~> 0.16.0"},
+      {:membrane_opus_plugin, "~> 0.17.1"},
+      {:membrane_aac_fdk_plugin, "~> 0.15.1"},
+      {:membrane_h264_plugin, "~> 0.7.2"},
```

Comment:
(if we no longer use parsers)
```suggestion
```

EDIT: we use it in tests, so let's add `only: :test`